src/components/staking/Undelegations.vue

```diff
@@ -26,6 +32,13 @@ export default {
   computed: {
     ...mapGetters([`address`, `network`])
   },
+  methods: {
+    filterOnlyPendingUndelegations(undelegations) {
```

Comment:
sweet! I agree though, that we should put this logic into the API to make the FE stupider